README.md

```diff
@@ -200,26 +200,49 @@ if (is_scalar($where)) {
 ### Router ###
 
 We provide you with a router if you want to match contentful elements directly through the url to a controller action
- of your app. Just register our Slug-Matcher with your [CMF-Routing-Chain](https://symfony.com/doc/current/cmf/components/routing/chain.html):
+ of your app. Just register our Slug-Matcher with your [CMF-Routing-Chain](https://symfony.com/doc/current/cmf/components/routing/chain.html)
+ 
+ You may choose between the caching Slug-Matcher with this service declaration:
+ 
+```yaml
+services: 
+ app.router.contentful:
+     class: BestIt\ContentfulBundle\Routing\CachingContentfulSlugMatcher
+     public: false
+     arguments:
+         - '@best_it_contentful.cache.pool.routing'
+         - '@contentful.delivery.contentful_client'
+         - '%best_it_contentful.controller_field%'
+         - '%best_it_contentful.routing_field%'
+         - '@best_it_contentful.routing.route_collection_response_parser'
+         - '@best_it_contentful.delivery.response_parser.default'
```

Comment:
please add at last the centralized alias for this: best_it_contentful.delivery.response_parser = best_it_contentful.delivery.response_parser.default

Reply:
Alias implemtented